octobot_trading/personal_data/orders/order.py

```diff
@@ -124,7 +124,7 @@ def update(self, symbol, order_id="", status=enums.OrderStatus.OPEN,
                current_price=constants.ZERO, quantity=constants.ZERO, price=constants.ZERO, stop_price=constants.ZERO,
                quantity_filled=constants.ZERO, filled_price=constants.ZERO, average_price=constants.ZERO,
                fee=None, total_cost=constants.ZERO, timestamp=None,
-               order_type=None, reduce_only=False, close_position=False, position_side=None, fees_currency_side=None,
+               order_type=None, reduce_only=None, close_position=None, position_side=None, fees_currency_side=None,
```

Comment:
Issue was that we were setting `self.reduce_only` everytime we were calling update since we couldn't check that the param was actually passed (same for close_position)

Reply:
Interesting